src/scripts/Providers/OSUI/Dropdown/VirtualSelect/AbstractVirtualSelect.ts

```diff
@@ -40,6 +42,13 @@ namespace Providers.OSUI.Dropdown.VirtualSelect {
 			}
 		}
 
+		// Method to close the Dropdown if it's opened
+		private _closeOpened(): void {
```

Comment:
Sorry to be picky about names, but this one could be better 😂 

Why not simply _close? 

Or do we need this method, instead of using the existing one, with this validation?

Reply:
@bmarcelino-fe this method is when the events `onorientationchange` and `onresize` is triggered. Can't be simply close because this method only do it if it's open. The existing method close is used on client action that is public and needs to pass parameters whith an async. After talking with @joselrio we decided to create a this method to be used only on the events.